presto-benchmark/src/test/java/com/facebook/presto/benchmark/MemoryLocalQueryRunner.java

```diff
@@ -80,6 +80,7 @@ public void installPlugin(Plugin plugin)
                 new DataSize(1, GIGABYTE),
                 new DataSize(2, GIGABYTE),
                 new DataSize(1, GIGABYTE),
+                Optional.empty(),
```

Comment:
this will cause compilation error

Reply:
Yeah let me fix, just found a couple of these. My maven has been acting up all week, I just cleaned out .m2. Let me push again.